src/components/NavBar/index.js

```diff
@@ -21,21 +24,61 @@ class NavBar extends Component {
     });
   }
 
+  handleResize = () => {
```

Comment:
Actually, I am thinking we can simplify all this logic via grid systems.

Here is an example: https://stackoverflow.com/a/65648737/2174742

Notice the `overflowedIndicator`

![Screen Shot 2021-03-22 at 14 36 14](https://user-images.githubusercontent.com/3696327/111998149-04c07900-8b1c-11eb-818a-052ef12ec194.png)